distributed/core.py

```diff
@@ -370,7 +534,10 @@ def stop(self):
                 # The demonstrator for this is Worker.terminate(), which
                 # closes the server socket in response to an incoming message.
                 # See https://github.com/tornadoweb/tornado/issues/2069
```

Comment:
https://github.com/tornadoweb/tornado/pull/2075 is fixed now released in v5 so we should be able to call listener.stop()

Reply:
Done